amplitude_python_sdk/v2/ChartAnnotationsApiClient.py

```diff
@@ -0,0 +1,79 @@
+"""Client implementation for the Amplitude V2 Annotations API."""
+
+from typing import Optional, Dict, Any
+
+import requests
+
+from .models.charts import ChartAnnotationsV2
+from ..common.exceptions import AmplitudeAPIException
+from ..common.utils import return_or_raise
+
+
+class ChartAnnotationsApiClient:  # pylint: disable=too-few-public-methods
+    """
+    See <https://developers.amplitude.com/docs/chart-annotations-api> for documentation.
+    """
+
+    def __init__(
+        self, api_key: str,
+        api_endpoint: str = "https://api2.amplitude.com/2/annotations",
+        annotation_id: Optional[str] = None
+    ):
+        self.api_key = api_key
+        self.api_endpoint = api_endpoint
+        self.annotation_id = annotation_id
+
+    def _construct_chart_request_data(
+        self, annotation: ChartAnnotationsV2
+    ) -> Dict[str, Any]:
+        """
+        Convert ChartAnnotationsV2 model to dictionary payload for create annotation post request
+        """
+
+        req_data = {
+            "api_key": self.api_key,
+            "annotation": annotation.dict(exclude_none=True),
```

Comment:
I converted model similar to [here](https://github.com/researchrabbit/amplitude-python-sdk/blob/main/amplitude_python_sdk/v2/client.py#L25-L39) but am wondering why JSON parser not necessary

Reply:
It's because of the `json=` param in `requests.post`: https://github.com/researchrabbit/amplitude-python-sdk/pull/7/files/0872dccacd062ea83daf7206a082b038e2105356#diff-c622d29f7583f7f686185f54f75ea3aef6b613577efe823309e37eee8fd65c50R48

which will automatically convert the input dictionary to JSON

Reply:
so it looks like there are parsing issues with datetime objects.  I'm getting `TypeError: Object of type date is not JSON serializable` when I test out the post request.   That's probably the same case with the [EventV2](https://github.com/researchrabbit/amplitude-python-sdk/blob/main/amplitude_python_sdk/v2/models/event.py#L63) model.